script/Arcueid/Day 8/08_00_ARC08_4.txt

```diff
@@ -92,42 +101,55 @@ And while her back was turned, I spoke up,
 [sha:63f7d9149af669fbb937f931567a31457242b19b]{
 -- Page 237, Offset 20262.
 -- なんとなく、そんな質問を投げかけた。
-I throw out that question haphazardly.
+I lob him a vague question.
 }
 [sha:af0066be85716c3392bf8a6055affe8919bc2db5]{
 -- Page 238, Offset 20263. Mods: @k, @e.
 -- 「なんだい突然に。
-"Where did that come from all of a sudden?
+"What are you so curious about all of a sudden?
 }
 [sha:88cfc862302769db5dfcde1eb25dd404e0307d0c]{
 -- Page 238, Offset 20264. Glued. Mods: @x.
 -- 北口デパート並みにヤバめなスポットなんてそうそうねえぞ。……しいていうならアレか。<櫛塚|くしづか>の廃病院じゃねえか？　二年前まで<珍走団|ヤンキーども>の集会場になってたものの、今じゃ誰もよりつかなくなったっていう」
-There's hardly any other spot as bad as the northern department store... Well there is one if you ask me. It has to be the abandoned hospoital in Kushizuka right? It used to be where all those yankees gathered until two years ago, but nowadays it's become completely deserted."
+// Leading space for glue
+ There aren't really any spots that are as dodgy as the department store by the north gate...
+ If you pressed me on it, I guess there's one?
+ The abandoned hospital in Kushizuka fits the bill, right?
+ Some <biker gang|y a n k e e s   > used to be based out of there up until two years ago,
+ but these days it's become the sort of place nobody wants to go near."
 }
 [sha:2a65846a6034d4bb0c3f67a131a569b9829de100]{
 -- Page 239, Offset 20265.
 -- 「櫛塚って二駅となりのか？　でも、廃墟が物騒なのは当然というか、普通、誰もよりつかないだろ」
-"Kushizuka is the one two stations away, right? But it's natural, or rather obvious that an abandoned building is dangerous, so of course no one would use it."
+"Kushizuka's the town two stops away, right?
+ Everyone knows abandoned buildings are dangerous though,
+ so it makes sense that nobody would go near it."
 }
 [sha:7c8b14270671ee09865e33b30cad2f72538294f6]{
 -- Page 240, Offset 20266.
 -- 「いや、だからさ。誰も居なくなったんだよ。ある日を境に、ぱったりと、そのヤンキーどもの姿がな。三十人近い行方不明者ってコトで、今でも捜索願が出されてる。
-"No, like I said, it 'became' deserted. Like those yankees, one day, out of the blue. Around thirty of them went missing, and even now the police are still searching for them.
+"No, you're missing the point.
+ All the people that were there just disappeared.
+ Something happened one day and bam, those yankees were never seen again.
```

Comment:
again, I think "yankees" just fundamentally means something different in English. I'd go with "punks" or something